scene/resources/text_paragraph.cpp

```diff
@@ -490,16 +504,18 @@ void TextParagraph::draw_line_outline(RID p_canvas, const Vector2 &p_pos, int p_
 
 	Vector2 ofs = p_pos;
 	if (TS->shaped_text_get_orientation(lines[p_line]) == TextServer::ORIENTATION_HORIZONTAL) {
-		ofs.y += TS->shaped_text_get_ascent(lines[p_line]);
+		ofs.y += TS->shaped_text_get_ascent(lines[p_line]) + spacing_top;
 	} else {
-		ofs.x += TS->shaped_text_get_ascent(lines[p_line]);
+		ofs.x += TS->shaped_text_get_ascent(lines[p_line]) + spacing_top;
 	}
 	return TS->shaped_text_draw_outline(lines[p_line], p_canvas, ofs, -1, -1, p_outline_size, p_color);
 }
 
 TextParagraph::TextParagraph(const String &p_text, const Ref<Font> &p_fonts, int p_size, const Dictionary &p_opentype_features, const String &p_language, float p_width, TextServer::Direction p_direction, TextServer::Orientation p_orientation) {
 	rid = TS->create_shaped_text(p_direction, p_orientation);
 	TS->shaped_text_add_string(rid, p_text, p_fonts->get_rids(), p_size, p_opentype_features, p_language);
+	spacing_top = p_fonts->get_spacing(Font::SPACING_TOP);
+	spacing_bottom = p_fonts->get_spacing(Font::SPACING_BOTTOM);
 	width = p_width;
 	dirty_lines = true;
```

Comment:
Nitpick but this one is redundant since it's already initialized to `true` in the header.